crates/infra/utils/src/codegen/common/file_system.rs

```diff
@@ -31,7 +32,8 @@ pub fn write_file(file_path: &Path, contents: &str) -> Result<()> {
 }
 
 pub fn verify_file(file_path: &Path, contents: &str) -> Result<()> {
-    let formatted = format_source_file(file_path, contents)?;
+    let unformatted = add_header(file_path, contents);
```

Comment:
I will minimize the diff, to make it easier to review. The main change here is to write the failed file to original file path, instead of a backup, to make it easier to debug what went wrong in the editor/local files, same as what we do with everything else.

While it can be split into a different PR, I think it is a minimal fix to make it easier to debug anything that goes wrong here, and I would like to keep it here for pragmatism if you are OK with the new version.